app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">Mavel_03</string>
+    <string name="s24">24</string>
+    <string name="s400">400</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.s400` appears to be unused
 Unused resources make applications larger and slow down builds.